src/sentry/conf/server.py

```diff
@@ -1357,7 +1358,7 @@ def create_partitioned_queues(name):
 SENTRY_METRICS_SKIP_INTERNAL_PREFIXES = []  # Order this by most frequent prefixes.
 
 # Metrics product
-SENTRY_METRICS_INDEXER = "sentry.sentry_metrics.indexer.mock.MockIndexer"
+SENTRY_METRICS_INDEXER = "sentry.sentry_metrics.indexer.postgres.PGStringIndexer"
```

Comment:
I don't really like the naming, I just wanted to make it different that the redis indexer, and I wasn't sure if we were ready to use this as the _actual_ string indexer yet